meta/store.go

```diff
@@ -129,6 +129,12 @@ func (s *Store) IDPath() string { return filepath.Join(s.path, "id") }
 
 // Open opens and initializes the raft store.
 func (s *Store) Open() error {
+	// Verify that no more than 3 peers.
+	// https://github.com/influxdb/influxdb/issues/2750
+	if len(s.peers) > 3 {
```

Comment:
What I mean is that for this to work, `peers` must include the guy who is joining, correct?